src/python/sortingview/gui/extensions/sortingcomparison/sortingcomparison.tsx

```diff
@@ -0,0 +1,25 @@
+import { LabboxExtensionContext } from '../../pluginInterface';
+
+const CellComponent = (record: any) => {
+    return (
+        <span>{record}</span>
+    );
+}
+
+export function activate(context: LabboxExtensionContext) {
```

Comment:
This is a metric that returns the best matching unit in sorting B corresponding to units in sorting A.